ProjectEureka/project-eureka-flutter
# PE-28 | Created profile onboarding.

**What I did:** 

- Created a page that allows new users to input their information. We can also reuse this page for updating the profile. 
- Created a blank page to redirect to upon valid submission. 

**How to test:** 

- Test each field to when blank to see if the validator is working. It should NOT redirect to a new page if invalid. 
- Fill in your information as normal and test to see if it will submit and redirect properly when everything is correct. Also make sure that in the terminal, there is a printout of the values you've inputted.
